pages/common/pdfposter.md

```diff
@@ -0,0 +1,11 @@
+# pdfposter
+
+> Convert a large-sheeted pdf into multiple A4 tiles for printing.
```

Comment:
Should we change it to A4 pages instead of tiles?

Reply:
yeah, that's a good idea